eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/api/EclairApi.scala

```diff
@@ -33,7 +33,9 @@ trait EclairApi {
 
   def close(id: ChannelId, spk: ScriptPubKey): Future[String]
 
-  def findRoute(nodeIdOrInvoice: Either[NodeId, LnInvoice]): Future[Vector[String]]
+  def findRoute(nodeId: NodeId ): Future[Vector[String]]
+
+  def findRoute(invoice: LnInvoice): Future[Vector[String]]
```

Comment:
Let's implement this in the `EclairApi` file. You should be able to do 

```scala

def findRoute(invoice: LnInvoice): Future[Vector[String]] = {
  findRoute(nodeId = invoice.nodeId)
}

```

The return type for this should probably also change to 

```scala 

def findRoute(): Future[Vector[LnRoute]]

```

but we can change that on a separate follow up PR